users/views.py

```diff
@@ -230,8 +230,8 @@ def post(self, request):
 
                 send_mail(
                     subject='[Trackhub Registry] Password reset',
-                    from_email='bilal@ebi.ac.uk',  # 'trackhub-registry@ebi.ac.uk',
-                    message='Hi, \nPlease use the link below to reset your password \n' + full_url,
+                    from_email='trackhub-registry@ebi.ac.uk',
+                    message='Hi' + user.username + ', \nPlease use the link below to reset your password: \n' + full_url,
```

Comment:
I think you want an extra space:

```suggestion
                    message='Hi ' + user.username + ', \nPlease use the link below to reset your password: \n' + full_url,
```